### PR TITLE
Feature/populate favorite button

### PR DESCRIPTION
**Pull request by Echory**
## Checklist
- [ ] Tests written where required
- [ ] All tests passing
- [ ] Unused code/prys/etc deleted
Testing Coverage at <percentage>%

## Changes Made
  
What (if any) features are you implementing?
- Implementing the ability to favorite a submission as an admin

What (if anything) did you refactor?
- Used refetch instead of holding state in hook
- Added stars to all musician cards
- Changed grid to flexbox for musician cards 

Were there any issues that arose?
- Had to call the submission ids a little differently than I thought and that held me up

Is there anything that you need from your teammate?
- Let me know if everything looks good!

Any other comments, questions, or concerns?

### CODE REVIEW

File: src/components/MusicianCard/MusicianCard.js [hunk 1]
```diff
@@ -1,10 +1,26 @@
...
+  const [favoriteSubmission] = useMutation(FAVORITE_SUBMISSION_ADMIN);
+
+  const selectFavorite = async (evt) => {
+    evt.preventDefault()
+    await favoriteSubmission({ variables: { submissionId: Number(submission.id), adminId:  Number(user.id)} })
+    refetch()
+  }
+
+
+  const showFavoriteButton = user && <img onClick={selectFavorite} src={submission.adminFavorite ? goldStar : starIcon } className='star-icon' />
```
Comment (mr-ryan12) [inline src/components/MusicianCard/MusicianCard.js:18]: What a creative way to style the stars!

**Review by aliroemhildt — APPROVED** [review]
this looks great, thanks Emma!

File: src/mutations.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      id
+      submissionId
+      adminId
+      favorite
+    }
+  }
+}
+`
+
+export {FAVORITE_SUBMISSION_ADMIN}
```
Comment (mr-ryan12) [inline src/mutations.js:17]: I know this was a headache, but excellent job getting it to work!

**Review by mr-ryan12 — APPROVED** [review]
Many thanks Emma! This looks great!

**Comment by aliroemhildt** [comment]
Note: this failing CI test is fixed on Ryan's active branch and will be resolved when we merge that in today

**Pull request merged**